scapy/layers/inet6.py

```diff
@@ -60,6 +60,12 @@
     XIntField,
     XShortField,
 )
+from scapy.fields import BitEnumField, BitField, ByteEnumField, ByteField, \
```

Comment:
Sorry but this is duplicated. Looks like a rebase mistake.

Reply:
Sorry for that!